catalog/app/containers/Admin/Buckets/Buckets.tsx

```diff
@@ -1266,23 +1265,25 @@ const columns = [
   },
 ]
 
+type BucketRow = Pick<
+  Model.GQLTypes.BucketConfig,
+  'name' | 'title' | 'description' | 'lastIndexed' | 'iconUrl'
+>
+
 interface CRUDProps {
   bucketName?: string
 }
 
 function CRUD({ bucketName }: CRUDProps) {
   const { bucketConfigs: rows } = GQL.useQueryS(BUCKET_CONFIGS_QUERY)
-  const [filter, setFilter] = React.useState('')
-  const filtered = React.useMemo(
-    () =>
-      filter
-        ? rows.filter(({ name, title }) =>
-            (name + title).toLowerCase().includes(filter.toLowerCase()),
-          )
-        : rows,
-    [filter, rows],
-  )
-  const ordering = Table.useOrdering({ rows: filtered, column: columns[0] })
+  const filtering = Table.useFiltering<BucketRow>({
```

Comment:
afaict you don't have to provide an explicit type arg (`<BucketRow>`) here, bc `rows` var is typed, and so `useFiltering` should infer the type of the argument just fine

Reply:
Yes, it works without type args. It works for type checking, but resulting types are barely readable for my taste.